lib/task-data/tasks/run-emc-diag.js

```diff
@@ -0,0 +1,26 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+
+module.exports = {
+    friendlyName: 'Run EMC Diag',
+    injectableName: 'Task.Os.Run.Emc.Diag',
+    implementsTask: 'Task.Base.Os.Install',
+    options: {
+        kernelFile: 'vmlinuz-4.1.15-diag',
+        initrdFile: 'ramfs.lzma',
+        kernelUri: '{{ api.server }}/common/{{ options.kernelFile }}',
+        initrdUri: '{{ api.server }}/common/{{ options.initrdFile }}',
```

Comment:
@anhou  This follows the same pattern as `bootstrap-ubuntu.js`.  https://github.com/RackHD/on-tasks/blob/master/lib/task-data/tasks/bootstrap-ubuntu.js

Should we change all of these task definitions?  Is this a gating issue for you?